src/pyosmeta/models/github.py

```diff
@@ -69,6 +76,17 @@ class Labels(BaseModel):
     description: Optional[str] = None
     color: Optional[str] = None
     default: Optional[bool] = None
+    type: Optional[LabelType] = None
+
+    @model_validator(mode="before")
```

Comment:
It's been so long since i've dug into this code. let's flesh out the docstrings while this is still fresh. 

can you please add a second "paragraph" that better describes what we are actually parsing here and why as a reminder (for me an our future selves)?

Reply:
I clarified the docstrings in https://github.com/pyOpenSci/pyosMeta/commit/b9868eb69224b03a282d27cdc03c28e3ab34ee85